assets/runtime/functions

```diff
@@ -817,26 +817,24 @@ gitlab_configure_project_features() {
 
 gitlab_configure_registry(){
   echo "Configuring gitlab::registry..."
-  if [[ $GITLAB_REGISTRY_PORT != 443 ]]; then
-    update_template ${GITLAB_CONFIG} \
-      GITLAB_REGISTRY_ENABLED \
-      GITLAB_REGISTRY_DIR \
-      GITLAB_REGISTRY_HOST \
-      GITLAB_REGISTRY_PORT \
-      GITLAB_REGISTRY_API_URL \
-      GITLAB_REGISTRY_KEY_PATH \
-      GITLAB_REGISTRY_ISSUER
-  else
-    GITLAB_REGISTRY_PORT="" # Sets Registry Port to empty to set the key empty in the  .gitlab-ci.yml
-    update_template ${GITLAB_CONFIG} \
-      GITLAB_REGISTRY_ENABLED \
-      GITLAB_REGISTRY_DIR \
-      GITLAB_REGISTRY_HOST \
-      GITLAB_REGISTRY_PORT \
-      GITLAB_REGISTRY_API_URL \
-      GITLAB_REGISTRY_KEY_PATH \
-      GITLAB_REGISTRY_ISSUER
+
+  if [[ $GITLAB_REGISTRY_PORT == 443 ]]; then
```

Comment:
Could we change `$GITLAB_REGISTRY_PORT` to `${GITLAB_REGISTRY_PORT}` to be more consistent. I know this is an error of my own.